CONTRIBUTING.md

```diff
@@ -8,6 +8,8 @@ Hi! Thanks for checking out Julia. If you have questions or problems, the [Julia
 
 * [Fork Julia](https://github.com/JuliaLang/julia/fork).
 
+* Sign up for [Travis](https://travis-ci.org/profile/) and switch on your julia fork.
```

Comment:
It can still be mentioned as an optional step, if people want to test privately on their own fork first and don't happen to have access to a Linux machine/VM themselves.